assets/js/linear_algebra/lib.js

```diff
@@ -776,6 +776,77 @@ function text_table_to_list(texts,
 
   return list_of_texts;
 }
+function text_matrix_to_list(coord_texts, coord,
+                             size, coord_keys, brack_key) {
+    // print vector u: vec_texts = ['u =', 'v =', ...], coord_texts = [[u.x, u.y], [v.x, v.y],
+                   //  keys = [[xu, yu,], [xv, yv] ...], 
+                  // coord = [x, y] coord to start printing, size = font_size;
+
+  let size_of_space = coord_texts[0].length,
+      numb_of_vector =  coord_texts.length,
+      matrix_w = 0.62 * size/14 * numb_of_vector,
+      matrix_h = 0.36 * size/14 * size_of_space,
+
+      texts_list = [],
+      lines_list = [],
+      
+      cols_list = [],
+      rows_list = [];
+  
+  // text
+  cols_list.push(coord[0]);
+  for (i = 1; i < numb_of_vector; i++) {
+    cols_list.push(cols_list[0] + 0.6 * size/14 * i);
+  }
+
+  rows_list.push(coord[1] - matrix_h/2 + 0.225 * size/14);
+  for (i = 1; i < size_of_space; i++) {
+    rows_list.push(rows_list[0] + 0.36 * size/14 * i);
+  }
+
+  for (i = 0; i < numb_of_vector; i++) {
+    for (j = 0; j < size_of_space; j++) {
+      texts_list.push({text: coord_texts[i][j], x: cols_list[i],
+                       y: rows_list[j], font_size: size,
+                       text_opacity: 1.0, key: coord_keys[i][j]});
+    }
+  };
```

Comment:
can we reuse `text_table_to_list()` instead of these lines of code?

Reply:
nice.

Reply:
done :)